frontend/packages/console-app/src/redux/reducers/cloud-shell-selectors.ts

```diff
@@ -7,3 +10,8 @@ export const isCloudShellExpanded = (state: RootState): boolean =>
 
 export const isCloudShellActive = (state: RootState): boolean =>
   !!state.plugins?.console?.[cloudShellReducerName]?.isActive;
+
+export const getCloudShellCommand = (state: RootState): boolean =>
+  state.plugins?.console?.[cloudShellReducerName]?.command;
+
+export const useCloudShellCommandSelector = (): string | null => useSelector(getCloudShellCommand);
```

Comment:
Not sure there's value in keeping this vs using `useSelector(getCloudShellCommand)` which is what's done with the other selectors.